src/symbols.js

```diff
@@ -29,8 +35,10 @@ export function getVariables(formulaText) {
   const variables = formulaText
     // Replace the alternate multiplication symbol 'x' with a non-alphanumeric character
     .replace(/(^|\W)x(\W|$)/, ' ')
+    // Replace functions with a non-alphanumeric character
+    .replace(/max\s*\(/, ' ')
```

Comment:
It's great that you used a symbolic constant `FUNCTION_NAMES` to define the supported variables. Let's use this here to make it more future-proof!